pkg/store/kotsstore/version_store.go

```diff
@@ -71,6 +71,10 @@ func (s *KOTSStore) IsRollbackSupportedForVersion(appID string, sequence int64)
 		return false, errors.Wrap(err, "failed to load kots app from contents")
 	}
 
+	if util.IsEmbeddedCluster() {
```

Comment:
why not at the very beginning of the function? :)

Reply:
sure, why not

Reply:
actually, "because that might impact behavior more than where it is now"

maybe moving it to the beginning means that 'can't get app' errors are surfaced differently